requirements.txt

```diff
@@ -5,6 +5,7 @@
 # doc/requirements.rst
 numpy>=1.17.4
 PySide2
-scikit-surgerycore
-scikit-surgeryutils>=0.6.2
-scikit-surgeryarucotracker>=0.1.1
+scikit-surgerycore>=0.1.7
+scikit-surgeryutils>=1.2.0
+scikit-surgeryarucotracker>= 0.2.7
+scikit-surgeryopencvcpp
```

Comment:
@mxochicale I don't understand why scikit-surgeryopencvcpp has been added here. I don't think it's needed for the tutorial and is causing CI failures on some platforms. Can you please check and remove if possible?

Reply:
Was not sure about its version and how it was installed by default by other requirements so I added it, also because of this:
```
  Using cached shiboken2-5.12.0-5.12.0-cp35.cp36.cp37-abi3-manylinux1_x86_64.whl (175 kB)
INFO: This is taking longer than usual. You might need to provide the dependency resolver with stricter constraints to reduce runtime. See https://pip.pypa.io/warnings/backtracking for guidance. If you want to abort this run, press Ctrl + C.
Collecting PySide2
  Using cached PySide2-5.11.1-5.11.1-cp35.cp36.cp37-abi3-manylinux1_x86_64.whl (166.5 MB)
  Using cached PySide2-5.11.0-5.11.0-cp35.cp36-abi3-manylinux1_x86_64.whl (166.4 MB)

Pip subprocess error:
ERROR: Ignored the following versions that require a different python version: 5.14.0 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9; 5.14.1 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9; 5.14.2 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9; 5.14.2.1 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9; 5.14.2.2 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9; 5.14.2.3 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9; 5.15.0 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9
ERROR: Could not find a version that satisfies the requirement scikit-surgeryopencvcpp (from scikit-surgerycalibration) (from versions: none)
ERROR: No matching distribution found for scikit-surgeryopencvcpp
```
but I just removed it.

Reply:
OK, thanks, I think I understand now. scikit-surgeryopencvcpp is a requirement of scikit-surgerycalibration which is itself a requirement of scikit-surgeryutils. At some point we should aim to do some refactoring because there's a lot of code being pulled in here that isn't needed for the tutorial. But we'll have to leave it for now.